src/herbpy/action/rogue.py

```diff
@@ -0,0 +1,321 @@
+import logging, openravepy, prpy 
+from prpy.action import ActionMethod
+from prpy.planning.base import PlanningError
+from contextlib import contextmanager 
+from prpy.util import FindCatkinResource
+import numpy, cPickle, time, os.path
+
+logger = logging.getLogger('herbpy')
+
+@ActionMethod
+def Point(robot, focus, manip=None, render=False):
+    """
+    @param robot The robot performing the point
+    @param focus The 3-D coordinate in space or object 
+                 that is being pointed at
+    @param manip The manipulator to perform the point with. 
+                 This must be the right arm
+    @param render Render tsr samples during planning
+    """
+
+    #Pointing at an object
+    if type(focus) == openravepy.openravepy_int.KinBody:
+        focus_trans = focus.GetTransform()
```

Comment:
Can you clarify what you mean by that or point me to an example where that's done? I don't know how to do this..